pkg/volume/azure_dd/mounter.go

```diff
@@ -0,0 +1,179 @@
+/*
```

Comment:
follow the file name convention: the file that implement the plugin is named with the plugin's name.